Example/KnobGestureRecognizer/BpmControlView.swift

```diff
@@ -67,11 +67,9 @@ extension BpmControlView {
             AudioServicesPlaySystemSoundWithCompletion(1104, nil)
         }
         if #available(iOS 10.0, *) {
-            guard let feedbackGenerator = feedbackGenerator as? UISelectionFeedbackGenerator else {
-                return
-            }
-            feedbackGenerator.selectionChanged()
-            feedbackGenerator.prepare()
+            
+            let generator = UIImpactFeedbackGenerator(style: .medium)
```

Comment:
should we initialize generator every time when gesture event is occured? does initialization of generator have some performance hit?